pkg/codegen/docs/templates/resource.tmpl

```diff
@@ -54,17 +54,26 @@
 {{ template "constructor_args" }}
 {{ htmlSafe "{{% /choosable %}}" }}
 
-#### Resource Arguments
+## Resource Properties
+
+To learn more about resource properties and how to use them, see [Inputs and Outputs](https://www.pulumi.com/docs/intro/concepts/programming-model/#outputs) in the Programming Model docs.
```

Comment:
Yep, definitely.

Reply:
So I have been messing around trying to get the relative refs to work, but cannont get it to work because it keeps failing when trying to generate the {provider}/provider.md page because it sits at a different relative level than the other resources. So I don't think this is going to work.

```docs/reference/pkg/aws/provider/```
 vs.
```docs/reference/pkg/aws/s3/bucket/```

Reply:
`relref`'s need a leading `/`. Can you please try with that?

Reply:
Right -- always need a leading slash, otherwise Hugo may not be able to resolve relative URLs that have identical paths. Trailing slash should be omitted.

Reply:
OK, resolved the relrefs issue. Added the leading slash along with some fancy escaping to get it to render correctly. Thanks @justinvp for helping out.